test-it/karma/publicapi/TypeWaveform.Test.js

```diff
@@ -40,9 +41,12 @@ describe('Peek/Poke different datatypes', function () {
                 fraction: '0'
             },
             dt: 0,
-            Y: [] // eslint-disable-line id-length
+            Y: [], // eslint-disable-line id-length
+            attributes: { value: null, attributes: null }
         });
 
+        /*
```

Comment:
Supporting readJSON is complex and beyond what this PR is trying to get to. Since we do not really use the JSON API much, and there being more value in implementing the string formatter for Variants (they are used by the VTR test suite), I discussed with Milan and decided to disable these tests. The rest of the tests here give sufficient coverage.

Reply:
Was this intended to say writeJSON?

Reply:
Yup! Fixed it now.